src/config-import.js

```diff
@@ -65,10 +57,12 @@ export async function readConfig(configPath) {
       /** @type {IconData | undefined} */
       let icon
 
-      iconsZip = await yauzl.open(configPath)
-      const entries = await iconsZip.readEntries(MAX_ENTRIES)
-      for (const entry of entries) {
-        if (!entry.filename.match(/^icons\/([^/]+)$/)) continue
+      // we sort the icons by filename so we can group variants together
+      const iconEntries = entries
+        .filter((entry) => entry.filename.match(/^icons\/([^/]+)$/))
+        .sort((icon, nextIcon) => (icon.filename > nextIcon.filename ? 1 : -1))
```

Comment:
```suggestion
        .sort((icon, nextIcon) => icon.filename.localeCompare(nextIcon.filename))
```
Not a necessary change, but just sharing this string method if you don't know about it - can be handy for this kind of logic. However note that your current implementation is missing the case when the name is equal, which should return `0`, but this is ok given that it should not be possible for this array to include two entries with the same filename - just pointing out that you would normally need this to ensure a stable sort order I think.

Reply:
uhh I like local compare! prefer it much more that comparing strings as if they were 'numbers'